src/resources/elements/primeDesignSystem/ptooltip/ptooltip.ts

```diff
@@ -14,7 +14,7 @@ export class PTooltip {
   private tooltip: Instance;
 
   constructor(private element: Element) {
-    this.tooltip = tippy(this.element, {theme: "prime-design-system"});
+    this.tooltip = tippy(this.element, {theme: "prime-design-system", appendTo: document.body});
```

Comment:
See https://atomiks.github.io/tippyjs/v6/all-props/#appendto

Are you sure this works for all cases?  Should it be hard-coded as it is?  How about make it be a bindable string that is a selector, where the default is "body"?  But we need to make sure this works for all of the tooltip cases.

Reply:
could also be bindable to all three legal datatypes